plugins/PowerManagement/PowerManagementLinux.py

```diff
@@ -0,0 +1,70 @@
+'''
+Created on 10.01.2017
+
+@author: thopiekar
+'''
+
+import dbus
```

Comment:
It removes a dependency with relative ease. Would be very nice. Our build system is already quite complicated, as you know.